dfs/backup.go

```diff
@@ -82,7 +74,7 @@ func (dfs *DistributedFilesystem) Backup(data BackupInfo, w io.Writer) error {
 			return err
 		}
 		// load the images from this snapshot
-		glog.Infof("Preparing images for tenant %s", info.TenantID)
+		glog.V(2).Infof("Preparing images for tenant %s", info.TenantID)
```

Comment:
The backup logging was too chatty, so I took stuff down. I can bring stuff back up, but this (along with others) seemed like a message you would only ever care about if something went wrong, hence debug.